src/js/fc.js

```diff
@@ -893,4 +893,88 @@ const FC = {
     getSliderDefaults() {
         return this.DEFAULT_TUNING_SLIDERS;
     },
+
+    RATES_TYPE: {
+        BETAFLIGHT: 0,
+        RACEFLIGHT: 1,
+        KISS: 2,
+        ACTUAL: 3,
+        QUICKRATES: 4,
+    },
+
+    getCurrentRates() {
+
```

Comment:
To me is difficult to see this object here. In my opinion all this conversions must be done either:
- When reading the FC values using MSP directly. But maybe this complicates the code more.
- Or using an utility method, in the `RateCurve.js` library.